### PR TITLE
Add threshold for organizing imports with the asterisk (*) wildcard character

### LINKED ISSUE #964: Feature Request: organize imports with the asterisk (*) wildcard character
It would be great to organize imports from same package with a single asterisk (*), which makes the import section a little shorter.

In `eclipse`, there are two settings related to this feature:
1. `number of imports needed for .* (e.g. org.eclipse.*)`
2. `number of static imports needed for .* (e.g. java.lang.Math.*)`

Unfortunately, these options are not available in the eclipse formatter xml file ([available options](https://help.eclipse.org/oxygen/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjdt%2Fcore%2Fformatter%2FDefaultCodeFormatterConstants.html)).

### PR DESCRIPTION
**Pull request by snjeza**
Fixes #964 
Requires https://github.com/eclipse/eclipse.jdt.ls/pull/1407

Signed-off-by: Snjezana Peco <snjezana.peco@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -516,6 +516,18 @@
           "description": "The launch mode for the Java extension",
           "default": "Hybrid",
           "scope": "window"
+        },
+        "java.import.ondemandthreshold": {
```
Comment (fbricon) [inline package.json:520]: java.sources.organizeImports.starThreshold

File: package.json [hunk 2]
```diff
@@ -516,6 +516,18 @@
...
+          "type": "integer",
+          "description": "Specifies the number of imports added before a star-import declaration is used.",
+          "default": 99,
+          "scope": "window"
+        },
+        "java.import.staticondemandthreshold": {
```
Comment (fbricon) [inline package.json:526]: java.sources.organizeImports.staticStarThreshold

File: README.md [hunk 3]
```diff
@@ -157,6 +155,10 @@ New in 0.59.0:
 
   Default launch mode is `Hybrid`. Legacy mode is `Standard`
 
+  New in 0.60.0:
+* `java.import.ondemandthreshold`: Specifies the number of imports added before a star-import declaration is used, default is 99.
```
Comment (fbricon) [inline README.md:159]: java.sources.organizeImports.starThreshold

File: README.md [hunk 4]
```diff
@@ -157,6 +155,10 @@ New in 0.59.0:
...
+* `java.import.staticondemandthreshold`: Specifies the number of static imports added before a star-import declaration is used, default is 99.
```
Comment (fbricon) [inline README.md:160]: java.sources.organizeImports.staticStarThreshold

File: package.json (hunk 1 above)
Comment (snjeza) [reply package.json]: fixed.

File: README.md (hunk 3 above)
Comment (snjeza) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (snjeza) [reply README.md]: Fixed.

File: package.json (hunk 2 above)
Comment (snjeza) [reply package.json]: Fixed.

**Comment by snjeza** [comment]
@fbricon I have updated the PR.

**Comment by fbricon** [comment]
Thanks @snjeza

**Pull request merged**